doc: known_issues: Add new NCS 1.5 Thread known issues (#4035)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0

KRKNWK-9094: Possible deadlock in shell subsystem
  It is possible to cause a deadlock in the shell subsystem if the OpenThread commands are issued too fast.

>   **Workaround:** If possible avoid invoking next command before execution of the previous one has complete.

Reviewer: ```suggestion
  **Workaround:** If possible avoid invoking next command before execution of the previous one has completed.
```

> KRKNWK-9094: Possible deadlock in shell subsystem
>   It is possible to cause a deadlock in the shell subsystem if the OpenThread commands are issued too fast.
> 
>   **Workaround:** If possible avoid invoking next command before execution of the previous one has complete.

Reviewer 2: ```suggestion
KRKNWK-9094: Possible deadlock in shell subsystem
  Issuing OpenThread commands too fast might cause a deadlock in the shell subsystem.

  **Workaround:** If possible, avoid invoking a new command before execution of the previous one has completed.
```
